packages/graphiql/src/style.css

```diff
@@ -179,6 +179,12 @@ button.graphiql-tab-add > svg {
   position: relative;
 }
 
+/* The results editor wrapping container */
+.graphiql-container .graphiql-response .result-window {
+  position: relative;
+  flex: 1 1 100%;
```

Comment:
Could we just get away with `flex: 1` here? If there's no footer this is the only child so it will take up the full height anyways.

Reply:
great catch! fixed.